src/components/GameView/GameOverDialog.vue

```diff
@@ -102,6 +157,18 @@ export default {
         });
       });
     },
+    iconFromGameStatus(gameStatus){
+      switch (gameStatus){
+        case 'W':
+          return 'mdi-trophy'
+        case 'L':
+          return 'mdi-close-thick'
+        case 'I':
+          return 'mdi-account-clock'
+        case 'D':
+          return 'mdi-handshake-outline'
+      }
```

Comment:
Let's discuss the iconography. This is a reasonable first pass, but I think we can sort it out even clearer. I like the handshake for the draw. I think we want win and loss to look like obvious opposites, which I don't think trophy does well because what's the opposite of a trophy? 

So maybe we lean on the handshake and say it's thumbs up, thumbs down, and handshake?
![image](https://user-images.githubusercontent.com/6520704/220728135-8709921f-ea1a-4f87-9fc6-673770dee1a0.png)


![image](https://user-images.githubusercontent.com/6520704/220728197-6d627be4-e7dc-46c2-b3a5-09efc6b01e78.png)

There's also this possible option for stalemate/draw
![image](https://user-images.githubusercontent.com/6520704/220728351-66825bfd-13cd-4b3a-956e-b86edb4f8e69.png)


@seriouslysean @lihaobhsfer what do y'all think? I'd also pondered faces like smily, frown/dead for W/L. Penny for your thoughts

Reply:
I don't have much opinion here. As long as the size of the icons are consistent.

Reply:
I like the thumbs instead of the other ones.